quickevent/app/plugins/Event/src/connectdbdialogwidget.ui

```diff
@@ -33,24 +33,24 @@
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
-       <number>5</number>
+       <number>4</number>
       </property>
       <property name="leftMargin">
-       <number>3</number>
+       <number>9</number>
       </property>
       <property name="topMargin">
        <number>3</number>
       </property>
       <property name="rightMargin">
-       <number>3</number>
+       <number>9</number>
       </property>
       <property name="bottomMargin">
        <number>3</number>
       </property>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
-         <string>E&amp;vent ID</string>
+         <string>Open recent &amp;event if it is present in the database</string>
```

Comment:
`Event name` is enough, explanation can be set in edit-box tool-tip

Reply:
label - `Event ID`
placeholder - deleted
added tooltip